mobileapp/app.json

```diff
@@ -1,32 +1,4 @@
 {
-  "expo": {
-    "name": "mobileapp",
-    "slug": "mobileapp",
-    "version": "1.0.0",
-    "orientation": "portrait",
-    "icon": "./assets/icon.png",
-    "splash": {
-      "image": "./assets/splash.png",
-      "resizeMode": "contain",
-      "backgroundColor": "#ffffff"
-    },
-    "updates": {
-      "fallbackToCacheTimeout": 0
-    },
-    "assetBundlePatterns": [
-      "**/*"
-    ],
-    "ios": {
-      "supportsTablet": true
-    },
-    "android": {
-      "adaptiveIcon": {
-        "foregroundImage": "./assets/adaptive-icon.png",
-        "backgroundColor": "#FFFFFF"
-      }
-    },
-    "web": {
-      "favicon": "./assets/favicon.png"
-    }
-  }
+  "name": "mobileapp",
```

Comment:
Może dajmy nazwę porządnie na LN Payments

Reply:
to jest nazwa modułu a nie projektu

Reply:
no generalnie zmienić nazwę, gdziekolwiek jest, bo teraz wyświetla się tylko mobileapp